src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrUpdater.cs

```diff
@@ -304,6 +309,11 @@ private async Task<bool> UpdateRepositoryRecursively(
 
         foreach (VmrDependencyUpdate update in updates)
         {
+            if (update.Mapping.DisableSynchronization)
```

Comment:
Technically a similar `if` should go into `VmrInitializer`. Just for completness

Reply:
I would not add tests for that for the VmrInitializer though

Reply:
Ah right, will add it. Also will need to modify the test so the repo sync gets disabled after its initialized

Reply:
Ah, in that case let's not do that

Reply:
It's call sync disabled, not initialization disabled, so let's just not do it

Reply:
that was kind of my original thought, that the repo should still get initialized